pjsip/include/pjsua-lib/pjsua.h

```diff
@@ -6583,6 +6684,9 @@ PJ_DECL(pjsua_buddy_id) pjsua_buddy_find(const pj_str_t *uri);
 PJ_DECL(pj_status_t) pjsua_buddy_get_info(pjsua_buddy_id buddy_id,
                                           pjsua_buddy_info *info);
 
+PJ_DECL(pj_status_t) pjsua_buddy_get_blf_info(pjsua_buddy_id buddy_id,
```

Comment:
Need documentation for all the APIs (also below).
Missing `pjsua_buddy_del_blf()`.